vm/runtime.ts

```diff
@@ -706,14 +698,7 @@ module J2ME {
      *      higher priority thread is scheduled to run.
      */
     static scheduleRunningContext(ctx: Context) {
```

Comment:
Nit: the comment on this function still says a thread with a higher priority preempts the current thread, which is no longer the case, so that comment should be updated.